Automattic/woocommerce-payments
# Remove product scheduling from woocommerce_save_product_variation hook and prevent publish button spam

Fixes #3192 

#### Changes proposed in this Pull Request

This removes the scheduling callback from the `woocommerce_save_product_variation` hook to avoid duplicate products in Stripe when publish is selected multiple times.

As far as I can tell, it should no longer be necessary to trigger this on both `save_post` and `woocommerce_save_product_variation` hooks since we were doing this to get around a price update issue and we are now creating price data on the fly when creating new subscriptions.


#### Testing instructions

1. Create a new variable subscription product, with multiple variations
2. Click the "Publish" button in quick succession multiple times
3. On `develop` you will see multiple instances of the same variation in Stripe, equal to the number of clicks from step 2
4. On this branch there should only be one instance of each variation in Stripe
5. Try to update the product name.
6. Ensure the update is reflected correctly in Stripe


**Post merge**
